Doc/library/abc.rst

```diff
@@ -186,15 +186,15 @@ The :mod:`abc` module also provides the following decorator:
 
       class C(ABC):
           @abstractmethod
-          def my_abstract_method(self, ...):
+          def my_abstract_method(self, arg1):
```

Comment:
```suggestion
          def my_abstract_method(self, arg):
```

Reply:
I've used explicitly different names to highlight that they are different.